### PR TITLE
STY: fstrings doc/make.py

### PR DESCRIPTION
**Pull request by ShaharNaveh**
- [x] ref #29547
- [x] tests added / passed
- [x] passes `black pandas`
- [x] passes `git diff upstream/master -u -- "*.py" | flake8 --diff`
- [ ] whatsnew entry

### CODE REVIEW

File: doc/make.py [hunk 1]
```diff
@@ -76,25 +76,23 @@ def _process_single_doc(self, single_doc):
...
         else:
             raise ValueError(
-                (
-                    "--single={} not understood. Value should be a "
-                    "valid path to a .rst or .ipynb file, or a "
-                    "valid pandas object (e.g. categorical.rst or "
-                    "pandas.DataFrame.head)"
-                ).format(single_doc)
+                f"--single={single_doc} not understood. "
+                f"Value should be a valid path to a .rst or .ipynb file, "
```
Comment (WillAyd) [inline doc/make.py]: Shouldn't need the `f` prefix on lines where not used

File: doc/make.py [hunk 2]
```diff
@@ -290,15 +284,14 @@ def zip_html(self):
...
     cmds = [method for method in dir(DocBuilder) if not method.startswith("_")]
 
+    joined = ",".join(cmds)
     argparser = argparse.ArgumentParser(
-        description="pandas documentation builder",
-        epilog="Commands: {}".format(",".join(cmds)),
+        description="pandas documentation builder", epilog=f"Commands: {joined}",
     )
+
+    joined = ", ".join(cmds)
```
Comment (WillAyd) [inline doc/make.py]: Don't need to redeclare this

**Review by WillAyd — CHANGES_REQUESTED** [review]
Looks pretty good minor comments

File: doc/make.py (hunk 2 above)
Comment (MomIsBestFriend) [reply doc/make.py]: It's not redeclaration. one of the ```joined``` is with a space and one is without.

```joined = ",".join(cmds)```
```joined = ", ".join(cmds)```

**Comment by jreback** [comment]
thanks @MomIsBestFriend

**Pull request merged**